azkaban-common/src/main/java/azkaban/utils/AbstractMailer.java

```diff
@@ -26,6 +26,7 @@
   private boolean usesAuth;
 
   private String mailHost;
+  private String mailPort;
```

Comment:
Why not have mailPort as int since your are already parsing while calling t.connect ?